Forms/frmSplash.ACF

```diff
@@ -112,1121 +112,26 @@ Begin Form
                 0xe12fa329ea7bc540ae94cf4ebf07048a
             End
             Begin
-                Begin Image

```

Comment:
Please can you put an image of some sort back in. It looks awful without it. Perhaps https://favpng.com/png_view/fork-branching-source-code-version-control-fork-png/mce0WrLp